src/LESOPDRACHT.js

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+
+console.log( "Script is running" )
+
+// 1. Data Fetchen
+
+// Create reference
+const list = document.getElementById( 'list-of-users' );
+
+// const fetchData = async () => {}
+async function fetchData( num ) {
+    const URI = 'https://jsonplaceholder.typicode.com/'
+    const ENDPOINT = 'users'
```

Comment:
Variabelnamen in camelCase schrijven
```suggestion
    const uri = 'https://jsonplaceholder.typicode.com/'
    const endpoint = 'users'
```